src/training/communicator_nccl.h

```diff
@@ -191,7 +191,7 @@ class NCCLCommunicator : public ICommunicator {
     groupEnd();
 
     mpiBarrier(); // (synchronize the log messages)
-    LOG(info, "[comm] NCCLCommunicator constructed successfully.");
+    LOG(debug, "[comm] NCCLCommunicator constructed successfully");
```

Comment:
As above, let's keep info. These log messages are not really useful when everything works, but useful if it didn't work. Good to have them in there by default in later error assessment, even when a user files issues.

Reply:
Done